.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: c
+compiler:
+  - clang
+  - gcc
+install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libarchive-dev nettle-dev dejagnu expect mcrypt
+script:
+  - rm -Rf ~/.gnupg # Skip gpg tests
```

Comment:
Otherwise gpg blocks asking for recipient as no DefaultRecipient is set in gpg.conf.

Ideally some commands to fetch some pubkeys should have been used and recipient explicitly stated. Something to add later.